DS_Assignment_6/Employee/EmployeeMain.java

```diff
@@ -0,0 +1,86 @@
+package Employee;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.TreeSet;
+
+
+public class EmployeeMain {
+
+	public static void main(String[] args) {
+		Set<Employee> set = new TreeSet<Employee>();
+		Scanner scan;
+		Employee employee;
+		List<Employee> list;
+
+		char ch;
+		try {
+			do {
+				scan = new Scanner(System.in);
+				System.out.println("\nOperations");
+				System.out.println("1. Add Employee");
+				System.out.println("2. Display by Natural Order");
+				System.out.println("3. Display by Name");
+				System.out.println("4. Display by Address");
+				System.out.println("5. Exit");
+
+				int choice = scan.nextInt();
+
+				switch (choice) {
+				case 1:
+					System.out.println("Enter employee id=");
+					int empid = scan.nextInt();
+					System.out.println("Enter name=");
+					scan.nextLine();
+					String name = scan.nextLine();
+					System.out.println("Enter Address=");
+					String addr = scan.nextLine();
+					employee = new Employee(empid, name, addr);
+
+					//adding employee to set
+					if (!set.add(employee)) {
+						System.out.println("Employee Already Exists");
+					} else {
+						System.out.println("Employee added");
+					}
+					break;
+					
+				case 2:
+					System.out.println(set);

```

Comment:
Where are you doing the natural sorting? as the per the option above case 2 should  display in natural order.